src/res_outcome_printer.ml

```diff
@@ -233,17 +233,19 @@ let printPolyVarIdent txt =
        )
      | Otyp_alias (typ, aliasTxt) ->
        Doc.concat [
+         Doc.lparen;
          printOutTypeDoc typ;
          Doc.text " as '";
-         Doc.text aliasTxt
+         Doc.text aliasTxt;
+         Doc.rparen
        ]
      | Otyp_constr (
         Oide_dot (Oide_dot (Oide_ident "Js", "Fn") , "arity0"), (* Js.Fn.arity0 *)
         [Otyp_constr (Oide_ident ident, [])] (* int or unit or string *)
        ) ->
         (* Js.Fn.arity0<int> -> (.) => int*)
         Doc.concat [
-          Doc.text "(.) => ";
+          Doc.text "(. ()) => ";
```

Comment:
I wonder if we should also support `(.) => int` when parsing type expressions. We do support it in non-type expressions.

Reply:
I did not know it was not supported. Sounds like it should be, to avoid confusion.

Reply:
> For the alias type, would you add this example:
> 
> ```rescript
> let alias1: [> rgb] as 'a
> let alias2: ([> rgb] as 'a)
> ```

Not entirely sure what you mean here. I can add something like this to `test/oprint/oprint.res`:

```rescript
let alias1: [> rgb] as 'a = #Red
let alias2: ([> rgb] as 'a) = #Green
```

but that wouldn't cause the outcome printer to produce an alias, just `[> rgb]`.

Reply:
This surprised me too, and I think it should.

But also, for the record, `refmt` will convert `(.)` into `(. ())` for value expressions and `(. ())` into `(. unit)` for type expressions. Perhaps we should run this through `refmt` to ensure it produces idiomatic code too?